Dockerfile

```diff
@@ -80,7 +80,7 @@ RUN curl -fSL "${FLATBUFFERS_ARCHIVE_BASE_URL}/${FLATBUFFERS_ARCHIVE_TAG}" -o fl
 
 # Same build environment as FlatBuffer
 
-ARG FLATCC_ARCHIVE_BASE_URL="https://api.github.com/repos/dvidelabs/flatcc/tarball/"
```

Comment:
The trailing slash is unnecessary and has been removed to keep the style consistent.